server.py

```diff
@@ -536,6 +536,53 @@ def bus_hail():
 
   return jsonify(hail)
 
-
+@app.route('/woah', methods=['POST'])
+def woah():
+  '''
+    Basic endpoint to catch/throw "the woah"
+    ---
+    tags:
+      - ASCII
+    parameters:
+    - in: body
+      schema:
+        id: woah
+        type: object
+        properties:
+          value:
+            required: True
+            type: string
+            example: 'catch the whoa!'
+            description: a command to tell whether or not to "catch the woah"
+    responses:
+      200:
+        description: A JSON object of us catching the woah
+        content:
+          text/json:
+            schema:
+              properties:
+                message:
+                  type: string
+                  example: '        😲
+                                  ✊|
+                                    |🤚
+                                   / \\
+                                  /    \\'
+                  description: Caught the woah!
+    
+  '''
+  value = request.form.get('value', '')
```

Comment:
I think we might want to use JSON instead of form data to make our APIs consistent. Many online HTTP APIs will also use JSON.

Reply:
i.e., use `request.get_json()` instead of `request.form`

Reply:
I've changed it to a GET request - so using `request.args.get('value')` now instead 👍